### ISSUE TITLE
Panic using auto-embeddings

### ISSUE THREAD

**Issue by carlosbaraza** [meilisearch/meilisearch#4361]
**Describe the bug**
After hours processing embeddings using the model `BAAI/bge-base-en-v1.5`, the process panicked.

```
...
[compose-up] ip-app-ip-meilisearch-1      | [2024-01-24T21:31:15Z DEBUG TimerFinished] extract_embeddings(), Elapsed=6404.51813205s
[compose-up] ip-app-ip-meilisearch-1      | [2024-01-24T21:31:15Z DEBUG TimerFinished] extract_vector_points(), Elapsed=78.855209ms
[compose-up] ip-app-ip-meilisearch-1      | [2024-01-24T21:33:16Z DEBUG TimerFinished] extract_embeddings(), Elapsed=121.243060556s
[compose-up] ip-app-ip-meilisearch-1      | [2024-01-24T21:33:16Z DEBUG TimerFinished] extract_vector_points(), Elapsed=75.742875ms
[compose-up] ip-app-ip-meilisearch-1      | [2024-01-24T21:53:33Z DEBUG TimerFinished] extract_embeddings(), Elapsed=6243.309116265s
[compose-up] ip-app-ip-meilisearch-1      | [2024-01-24T21:53:33Z DEBUG TimerFinished] extract_vector_points(), Elapsed=49.549459ms
[compose-up] ip-app-ip-meilisearch-1      | thread 'indexing-thread:3' panicked at 'Cannot start a runtime from within a runtime. This happens because a function (like `block_on`) attempted to block the current thread while the thread is being used to drive asynchronous tasks.', milli/src/update/index_documents/extract/extract_vector_points.rs:379:18
[compose-up] ip-app-ip-meilisearch-1      | note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
[compose-up] ip-app-ip-meilisearch-1      | [2024-01-24T21:53:33Z DEBUG TimerFinished] extract_vector_points(), Elapsed=52.161667ms
[compose-up] ip-app-ip-meilisearch-1      | thread 'indexing-thread:0' panicked at 'Cannot start a runtime from within a runtime. This happens because a function (like `block_on`) attempted to block the current thread while the thread is being used to drive asynchronous tasks.', milli/src/update/index_documents/extract/extract_vector_points.rs:379:18
[compose-up] ip-app-ip-meilisearch-1      | [2024-01-24T21:53:33Z DEBUG TimerFinished] extract_vector_points(), Elapsed=59.057167ms
[compose-up] ip-app-ip-meilisearch-1      | [2024-01-24T21:53:33Z DEBUG TimerFinished] extract_embeddings(), Elapsed=14.550042ms
[compose-up] ip-app-ip-meilisearch-1      | [2024-01-24T21:53:33Z DEBUG TimerFinished] extract_embeddings(), Elapsed=16.839ms
[compose-up] ip-app-ip-meilisearch-1      | Rayon: detected unexpected panic; aborting
[compose-up] ip-app-ip-meilisearch-1 exited with code 0
```

**To Reproduce**
I can't reproduce.

**Expected behavior**
Not to panic

**Meilisearch version:**
v1.6.0

**Additional context**
Running on Docker on OSX.

**Comment by carlosbaraza** [comment]
Don't worry @dureuill, I try not to panic.

**Comment by curquiza** [comment]
fixed by #4371, will be released in v1.6.1
